fitbenchmarking/core/results_output.py

```diff
@@ -601,18 +601,25 @@ def display_page(pathname, profile_instances_all_groups,
     :type layout: list of dcc or html components
     :param max_solvers: Maximum number of solvers that can be plotted
     :type max_solvers: int
+    :param run_id: The id for the run that is plotted in dash
+    :type run_id: str
 
     :return: The updated layout
     :rtype: html.Div
     """
 
     try:
-        _, group, plot, metric_str = pathname.split('/')
+        _, plot_id, group, plot, metric_str = pathname.split('/')
     except ValueError:
         return ("404 Page Error! Path does not have the expected shape. "
                 "Please provide it in the following form:  \n"
```

Comment:
Just had a one word change suggestion to the error msg.
```suggestion
        return ("404 Page Error! Path does not have the expected format. "
                "Please provide it in the following form:  \n"
```